<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Case Dossier Management</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <!-- 左侧案件列表 -->
   <widget class="QListWidget" name="case_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>180</width>
      <height>580</height>
     </rect>
    </property>
   </widget>
   <!-- 素材表格 -->
   <widget class="QTableWidget" name="material_table">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>680</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <!-- 中间产物表格 -->
   <widget class="QTableWidget" name="artifact_table">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>680</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <!-- 日志窗口 -->
   <widget class="QTextEdit" name="log_window">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>680</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <!-- 按钮 -->
   <widget class="QPushButton" name="button_add_material">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>500</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Material</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_delete_material">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>500</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Material</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_view_material">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>500</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>View Material</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add_artifact">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>500</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Artifact</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_delete_artifact">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>500</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Artifact</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
